c2pa/claim-templates/claim_create.json

```diff
@@ -0,0 +1,88 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "{{recorder}}",
+    "assertions": [
+        {
+            "label": "stds.schema-org.CreativeWork",
+            "data": {
+                "@context": "https://schema.org",
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "{{author_type}}",
+                        "credential": [
+                        ],
+                        "identifier": "{{author_identifier}}",
+                        "name": "{{author_name}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "c2pa.actions",
+            "data": {
+                "actions": [
+                    {
+                        "action": "c2pa.created",
+                        "when": "{{authenticated_asset_created_timestamp}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "stds.exif",
+            "data": {
+                "@context": {
+                    "exif": "http://ns.adobe.com/exif/1.0/"
+                },
+                "exif:GPSLatitude": "{{gps_latitude}}",
+                "exif:GPSLatitudeRef": "{{gps_latitude_ref}}",
+                "exif:GPSLongitude": "{{gps_longitude}}",
+                "exif:GPSLongitudeRef": "{{gps_longitude_ref}}",
+                "exif:GPSTimeStamp": "{{gps_timestamp}}"
+            }
+        },
+        {
```

Comment:
This block should only be present if at least one of `dc` or `Iptc4xmpExt` block is present.